packages/astro-embed-twitter/Tweet.css

```
@@ -8,3 +8,6 @@
 .twitter-tweet:not(.twitter-tweet-rendered) > :last-child {
 	margin-bottom: 0;
 }
+astro-embed-tweet {
```

Just did a quick test, and think this might the safest way to specify this:

```suggestion
.twitter-tweet.twitter-tweet-rendered {
```

This ensures that the `color-scheme` override only applies to the component when Twitter has actually rendered its JS. In testing I didn’t see issues with setting it on `astro-embed-tweet`, but just in case some of the default markup needs to inherit the page value, this avoids interfering with that.